lib/pagseguro/payment_request/serializer.rb

```diff
@@ -9,6 +9,8 @@ def initialize(payment_request)
       end
 
       def to_params
+        serialize_credentials(payment_request.credentials)
```

Comment:
for now, we can just delegate the credential object to the request and within request verify if any credential is setted or use global credentials.

```ruby
params[:credentials] = payment_request.credentials
```

ps. don't forget to remove `params[:credentials]` before send the request to the api.